### ISSUE TITLE
SEG FAULT

### ISSUE THREAD

**Issue by stramel** [atom/atom#4892]
**OS:** Ubuntu 14.04
**Atom:** 0.167.0-8f4555b

#### Steps to Reproduce

1. Open Atom with `--safe`
2. File > Open Folder...
3. Select a project
4. Open a file
5. Right-click Paste

Here is the crash dump of running it in `--safe` mode: http://pastebin.com/eKHMHCfq

**Comment by Bengt** [comment]
Does this also appear with v0.166.0 for you or is this a regression in the unreleased Atom v0.167.0?

**Comment by stramel** [comment]
@Bengt Good point! I am trying to get 0.166.0 installed and running but after running the build it doesn't want to launch even with `--safe`.

**Comment by Bengt** [comment]
Atom does not have an uninstaller. Perhaps you could make a backup of your preferences, [delete the old files manually](https://github.com/atom/atom/issues/2195#issuecomment-42917489) and then install again from source. You can also force everything to be recompiled by deleting the `build` folder in atom's source code directory.

**Comment by stramel** [comment]
I'm building from source, I shouldn't need an uninstaller, correct?

**Comment by Bengt** [comment]
If I am not mistaking, building from source only compiles Atom to the `build` directory. Installing copies it to `/usr/local/bin/` and starting it generates the files under `~/.atom/`. ... Are you following the [instructions](https://github.com/atom/atom/blob/master/docs/build-instructions/linux.md#instructions)?

**Comment by stramel** [comment]
@Bengt steps 1-4;

**Comment by Bengt** [comment]
Step 4 copies Atom to `/usr/local/bin/atom`, so try to delete the executable and run step 4 again. If that does not help, remove `build` and run step 3 - 4 again. If that does not help, run `git clean -f´ and run step 2 - 4 again. If that does not help, delete your clone and run steps 1 - 4 again.

**Comment by lisotton** [comment]
I'm facing a crash when right click in somewhere (tree view, tabs) and select some option.

I'm using Ubuntu 14.04 and just installed through the .deb package.

**Comment by dikamilo** [comment]
Same here, Atom crash on any context menu action on linux.

System: Elementary OS
Atom: 0.168 by deb package from atom.io

**Comment by Swistaku** [comment]
@Bengt I've this problem on 168, now I've installed 166 from https://github.com/atom/atom/releases and the problem doesn't occur

**Comment by mnquintana** [comment]
Same issue here. Installed from the .deb package at atom.io

**System:** Ubuntu 14.04.1LTS
**Atom:** 0.168.0

**Comment by Bengt** [comment]
I can reproduce this under Fedora 20:

```
$ apm -v
apm  0.117.0
npm  1.4.4
node 0.10.33
python 2.7.5
git 1.9.3
$ uname -r
3.17.7-200.fc20.x86_64
```

with Atom v0.167.0:

```
$ sudo yum localinstall https://github.com/atom/atom/releases/download/v0.167.0/atom.x86_64.rpm
$ atom -v
0.167.0
$ atom --safe
/usr/local/bin/atom: line 92:  6272 Segmentation fault      (core dumped) nohup "$ATOM_PATH" --executed-from="$(pwd)" --pid=$$ "$@" > "$DOT_ATOM_DIR/nohup.out" 2>&1
[6272:0108/171811:ERROR:browser_main_loop.cc(163)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
App load time: 146ms
MESA-LOADER: could not create udev device for fd 12
MESA-LOADER: could not create udev device for fd 12
ATTENTION: default value of option force_s3tc_enable overridden by environment.
ATTENTION: option value of option force_s3tc_enable ignored.
[6313:0108/171812:ERROR:sandbox_linux.cc(305)] InitializeSandbox() called with multiple threads in process gpu-process
[6325:0108/171812:INFO:renderer_main.cc(207)] Renderer process started
[6272:0108/171813:INFO:CONSOLE(64)] "Window load time: 1382ms", source: file:///usr/local/share/atom/resources/app/static/index.js (64)
[6439:0108/171831:INFO:renderer_main.cc(207)] Renderer process started
[6272:0108/171833:INFO:CONSOLE(64)] "Window load time: 1332ms", source: file:///usr/local/share/atom/resources/app/static/index.js (64)
Crash dump id: a49dcefd-41f8-4743-b01f-e2af06c30033
```

and with Atom v0.168.0 as well:

```
$ sudo yum localinstall https://github.com/atom/atom/releases/download/v0.168.0/atom.x86_64.rpm
$ atom -v
0.168.0
/usr/local/bin/atom: line 92:  6746 Segmentation fault      (core dumped) nohup "$ATOM_PATH" --executed-from="$(pwd)" --pid=$$ "$@" > "$DOT_ATOM_DIR/nohup.out" 2>&1
[6746:0108/172355:ERROR:browser_main_loop.cc(163)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
App load time: 130ms
MESA-LOADER: could not create udev device for fd 12
MESA-LOADER: could not create udev device for fd 12
ATTENTION: default value of option force_s3tc_enable overridden by environment.
ATTENTION: option value of option force_s3tc_enable ignored.
[6787:0108/172356:ERROR:sandbox_linux.cc(305)] InitializeSandbox() called with multiple threads in process gpu-process
[6799:0108/172356:INFO:renderer_main.cc(207)] Renderer process started
[6746:0108/172357:INFO:CONSOLE(64)] "Window load time: 1329ms", source: file:///usr/local/share/atom/resources/app/static/index.js (64)
[6901:0108/172410:INFO:renderer_main.cc(207)] Renderer process started
[6746:0108/172411:INFO:CONSOLE(64)] "Window load time: 1383ms", source: file:///usr/local/share/atom/resources/app/static/index.js (64)
Crash dump id: 0612701f-4e54-41da-9d86-d4e0e8a36d7f
```

**Comment by Bengt** [comment]
@Swistaku Thanks for the workaround, downgrading to v0.166.0 works for me, too:

```
$ sudo yum remove atom
$ sudo yum localinstall https://github.com/atom/atom/releases/download/v0.166.0/atom.x86_64.rpm
```

**Comment by stramel** [comment]
I am not seeing the problem anymore with the version I am running.

```bash
$ apm -v
apm 0.117.0
npm 1.4.4
node 0.10.33
python 2.7.6
git 1.9.1
$ uname -r
3.13.0-43-generic
```

**OS:** Ubuntu 14.04
**Atom:** 0.169.0-1bc9c2e
**tree-view:** 0.147.0
**tabs:** 0.61.0

**Comment by kevinsawicki** [comment]
I think this might have been fixed by the upgrade to Atom Shell 0.20.5 which had this commit,  https://github.com/atom/atom-shell/commit/609e3ec3ffe97fc3c08924ca1c8f6c6fe9e6158c

**Comment by kevinsawicki** [comment]
Is anyone still seeing this on 0.169 or above?

**Comment by Bengt** [comment]
No, 0.169.0 fixes it for me.

**Comment by mnquintana** [comment]
Fixed for me in 0.169.0 too!

**Comment by lagseeing** [comment]
Do i have to compile from source to get 0.169.0 ?
